glide.yaml

```diff
@@ -34,3 +34,7 @@ import:
   version: 91cfa479c814065e420cee7ed227db0f63a5854e
 - package: github.com/kiali/k-charted
   version: v0.2.0
+- package: github.com/jtblin/go-ldap-client
+  version: ^0.0.3
+- package: gopkg.in/ldap.v2
+  version: ^3.0.3
```

Comment:
I asked the team what they think of using ^-notation in our dependencies - https://groups.google.com/forum/#!topic/kiali-dev/REq8_d5vuJI

For now, I would say it would be safest to just fix on a specific version and take out the ^.

Reply:
Made the changes to this file to fix the version